ui/pages/home/home.component.js

```diff
@@ -756,6 +771,99 @@ export default class Home extends PureComponent {
     );
   }
 
+  renderOnboardingPopover = () => {
```

Comment:
Might be nice to extract this into its own component/file.  But no big deal if you think it'd be difficult.

Reply:
Since we're on the clock I'd prefer to minimize larger changes like this.  Good idea moving forward though!